### DISCUSSION TITLE
Nyt

### DISCUSSION THREAD

**Discussion by hrondonuwu** [SingularityLabs-ai/beyond-bard-mini#7] [Q&A]
When I tell it to summarize an article in new york times, it can't access the entire article. I've tried with different summary extension and it works.

**Comment by ishandutta2007** [comment]
@NarutoUchiha69420 we are using the public version of bard, so it's esentially in hands of google, not much is in our hands, can you show the app that works for you , maybe we should try to learn from it ?
